packages/language-server/src/plugins/svelte/SveltePlugin.ts

```diff
@@ -74,9 +71,9 @@ export class SveltePlugin
             (await prettier.resolveConfig(filePath, { editorconfig: true })) ||
             this.prettierConfig ||
             // Be defensive here because IDEs other than VSCode might not have these settings
-            (this.editorConfig && this.editorConfig.tabSize && {
-                tabWidth: this.editorConfig.tabSize,
-                useTabs: !this.editorConfig.insertSpaces,
+            (options && {
```

Comment:
We fall back to an empty object in the instantiation of `SveltePlugin` in the `server.ts`. When I tried disabled the prettier extension it doesn't fall back to the editor options.

Reply:
Oh right 😄 this must be changed.